tests/integration/files/conf/sub_minion

```diff
@@ -33,6 +34,3 @@ grains:
       - jamie
       - zoe
 
-ipc_mode: tcp
```

Comment:
Leave the ipc_mode setting as it was, sub minion was purposely using TCP, even on Linux.